README.md

```diff
@@ -1,8 +1,14 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+If you wish to clone this project, you will need to create environment variables for the dynamic connection file to function as intended. 
```

Comment:
Good start! Consider whether there's anything else you need to let users know if they want to locally run your server - do they need to clone it? Run any commands? :D